custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -79,17 +86,13 @@ def _init_mlops(self, params):
                 )
             )
 
-        self._mlops = (
-            MLOps()
-            .set_model_id(params["model_id"])
-            .set_deployment_id(params["deployment_id"])
-            .set_channel_config(monitor_settings)
-            .agent(
-                mlops_service_url=params["external_webserver_url"],
-                mlops_api_token=params["api_token"],
+        self._mlops.set_channel_config(monitor_settings)
```

Comment:
Do we need to worry about the following from the base class?
```
if self._supports_chat():
    self._mlops.set_channel_config("spooler_type=API")
```
https://github.com/datarobot/datarobot-user-models/pull/1110/files#diff-a69efcf5368c853d546210b873e714a134fe082c7130cf5e37571b0c3313ba4cR127-R128

Reply:
`_configure_mlops()` is only called when `_supports_chat()` is `False` so they won't conflict with each other. Would readability be improved if I rename `_configure_mlops()` to _configure_mlops_non_chat()` or something like that?

Reply:
Renaming would help with readability, maybe calling it `_configure_mlops_for_non_chat()`

Reply:
Done 👍